DC/5.7.0/digizuite_constants.dcl

```diff
@@ -502,4 +502,13 @@ resource digizuite_config download_destination_id {
     autolink {
         constant = 'DOWNLOADDESTINATIONID'
     }
+}
+
+resource digizuite_config DOWNLOAD_NAME_REPLACE_MASK_FALLBACK {
+    constant = 'DOWNLOAD_NAME_REPLACE_MASK_FALLBACK'
+    config_value = '[%AssetId%]_[%MediaFormatId%]'
+    description = 'Replace mask fallback for Download name'
+	autolink {
+		constant = 'DOWNLOAD_NAME_REPLACE_MASK_FALLBACK'
+	}
```

Comment:
You can delete the `autolink` property.
We already autolink based on the `constant` property by default for `digizuite_config` resources 😉